### DISCUSSION TITLE
Proper place to file issues

### DISCUSSION THREAD

**Discussion by Mikaka27** [mozilla-mobile/fenix#26566] [Q&A] [answered]
Hi,
I have a question to the fenix developers. I understand that the team is overwhelmed by number of issues.
I'm a software developer myself, and when filing an issue I think I can make an educated guess, whether it is a problem in fenix/android components/gecko.

If I see a problem that is only happening on mobile firefox, but I believe the problem is in gecko regardless, should I file it to the fenix repo or the bugzilla?

**Comment by kbrosnan** [comment, marked as answer]
We don't expect people to understand the split between the Gecko core and Fenix project work. However if you try to reproduce the bug on [Geckoview Example Browser](https://firefox-ci-tc.services.mozilla.com/tasks/index/gecko.v2.mozilla-central.latest.mobile/android-aarch64-gcp-opt) and if it reproduces there then it is some sort of Gecko bug and is best filed in Bugzilla's Geckoview or Core component.

On the build artifact page download and install `geckoview_example.apk`. Note that to update it you need to uninstall, download a fresh version and install using Android's unknown app install. It uses a signature that is generated at build time so pave over installs will be blocked by Android because of a signature mismatch.

**Comment by Mikaka27** [comment]
Sorry I cannot find the artifact page, could you post a link? The link you posted is for status of some build task.

**Reply by kbrosnan** [reply]
Should have been https://firefox-ci-tc.services.mozilla.com/tasks/index/gecko.v2.mozilla-central.latest.mobile/android-aarch64-gcp-opt. I have updated my answer.

**Comment by Mikaka27** [comment]
Thank you for the information, I have what I need 😀
